docs/src/guides/storage.md

```diff
@@ -0,0 +1,37 @@
+# Configuring storage
+
+When using LanceDB OSS, you can choose where to store your data. The tradeoffs between different storage options are discussed in the [storage concepts guide](../concepts/storage.md). This guide shows how to configure LanceDB to use different storage options.
+
+## Object Stores
+
+LanceDB OSS supports object stores such as AWS S3 (and compatible stores), Azure Blob Store, and Google Cloud Storage. Which object store to use is determined by the URI scheme of the dataset path. For example, `s3://bucket/path` will use S3, `az://bucket/path` will use Azure, and `gs://bucket/path` will use GCS. In most cases, when running in the respective cloud and permissions are set up correctly, no additional configuration is required.
```

Comment:
The object store would have credentials specified by environment variables, no doubt. But for a new user, how would they go about configuring the database once they `pip install lancedb` or `npm install vectordb`? Is the only action on their part to specify `db = lancedb.connect("s3://bucket/path")`?

Reply:
It would still help to have the snippet of code showing how the DB connection is different from a local one.